src/components/Settings/SonarrModal/index.tsx

```diff
@@ -75,48 +82,56 @@ const messages = defineMessages('components.Settings.SonarrModal', {
   animeTags: 'Anime Tags',
   notagoptions: 'No tags.',
   selecttags: 'Select tags',
+  overrideRules: 'Override Rules',
+  addrule: 'New Override Rule',
 });
 
-interface TestResponse {
-  profiles: {
-    id: number;
-    name: string;
-  }[];
-  rootFolders: {
-    id: number;
-    path: string;
-  }[];
+interface SonarrTestResponse extends DVRTestResponse {
   languageProfiles:
     | {
         id: number;
         name: string;
       }[]
     | null;
-  tags: {
-    id: number;
-    label: string;
-  }[];
-  urlBase?: string;
 }
 
 interface SonarrModalProps {
   sonarr: SonarrSettings | null;
   onClose: () => void;
   onSave: () => void;
+  overrideRuleModal: { open: boolean; rule: OverrideRule | null };
+  setOverrideRuleModal: ({
+    open,
+    rule,
+    testResponse,
+  }: {
+    open: boolean;
+    rule: OverrideRule | null;
+    testResponse: DVRTestResponse;
+  }) => void;
 }
 
-const SonarrModal = ({ onClose, sonarr, onSave }: SonarrModalProps) => {
+const SonarrModal = ({
+  onClose,
+  sonarr,
+  onSave,
+  overrideRuleModal,
+  setOverrideRuleModal,
+}: SonarrModalProps) => {
   const intl = useIntl();
+  const { data: rules, mutate: revalidate } =
+    useSWR<OverrideRuleResultsResponse>('/api/v1/overrideRule');
   const initialLoad = useRef(false);
   const { addToast } = useToasts();
   const [isValidated, setIsValidated] = useState(sonarr ? true : false);
   const [isTesting, setIsTesting] = useState(false);
-  const [testResponse, setTestResponse] = useState<TestResponse>({
+  const [testResponse, setTestResponse] = useState<SonarrTestResponse>({
```

Comment:
Why is there a difference in consistency for test response in sonarr modal compared to radarr modal